test/lein_tools_deps/plugin_test.clj

```diff
@@ -0,0 +1,9 @@
+(ns lein-tools-deps.plugin-test
+  (:require [clojure.test :refer :all]
+            [lein-tools-deps.plugin :as plugin]))
+
+; The mere presence of this file means that `lein test` will trigger a compilation
+; of lein-tools-deps.plugin and at least we can know if it builds successfully.
+
+(deftest hello-travis
```

Comment:
This simple test can go away once additional tests are added, but I suggest to keep it here to validate, from the travis logs, that something is happening :-)